quality.gemspec

```diff
@@ -70,5 +70,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency('pronto-flake8')
   s.add_development_dependency('pronto-reek')
   s.add_development_dependency('pronto-rubocop')
+  s.add_development_dependency('pronto-punchlist')
```

Comment:
Gemspec/OrderedDependencies: Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `pronto-punchlist` should appear before `pronto-rubocop`.